[2023-09-04T06:02:12.762+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-12T00:00:00+00:00 [queued]>
[2023-09-04T06:02:12.783+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-12T00:00:00+00:00 [queued]>
[2023-09-04T06:02:12.783+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:02:12.814+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-03-12 00:00:00+00:00
[2023-09-04T06:02:12.825+0000] {standard_task_runner.py:57} INFO - Started process 5200 to run task
[2023-09-04T06:02:12.828+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-03-12T00:00:00+00:00', '--job-id', '380', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmpes7umdh6']
[2023-09-04T06:02:12.832+0000] {standard_task_runner.py:85} INFO - Job 380: Subtask delete_data_from_table
[2023-09-04T06:02:12.958+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-12T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:02:13.135+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-12T00:00:00+00:00'
[2023-09-04T06:02:13.138+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-03-12' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:02:13.155+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:13.174+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:02:13.195+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-03-12' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:02:13.204+0000] {sql.py:427} INFO - Rows affected: 0
[2023-09-04T06:02:13.226+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220312T000000, start_date=20230904T060212, end_date=20230904T060213
[2023-09-04T06:02:13.524+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:02:13.565+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-04T06:04:57.644+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-12T00:00:00+00:00 [queued]>
[2023-09-04T06:04:57.679+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-12T00:00:00+00:00 [queued]>
[2023-09-04T06:04:57.681+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-04T06:04:57.718+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): delete_data_from_table> on 2022-03-12 00:00:00+00:00
[2023-09-04T06:04:57.728+0000] {standard_task_runner.py:57} INFO - Started process 6509 to run task
[2023-09-04T06:04:57.734+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v04', 'delete_data_from_table', 'scheduled__2022-03-12T00:00:00+00:00', '--job-id', '926', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres_operator.py', '--cfg-path', '/tmp/tmp9livjgsn']
[2023-09-04T06:04:57.738+0000] {standard_task_runner.py:85} INFO - Job 926: Subtask delete_data_from_table
[2023-09-04T06:04:57.883+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v04.delete_data_from_table scheduled__2022-03-12T00:00:00+00:00 [running]> on host 10bdc2e98230
[2023-09-04T06:04:58.150+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mohghaff' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v04' AIRFLOW_CTX_TASK_ID='delete_data_from_table' AIRFLOW_CTX_EXECUTION_DATE='2022-03-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-12T00:00:00+00:00'
[2023-09-04T06:04:58.154+0000] {sql.py:274} INFO - Executing: 
            delete from dag_runs where dt = '2022-03-12' and dag_id = 'dag_with_postgres_operator_v04';
        
[2023-09-04T06:04:58.179+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:58.196+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-04T06:04:58.221+0000] {sql.py:418} INFO - Running statement: 
            delete from dag_runs where dt = '2022-03-12' and dag_id = 'dag_with_postgres_operator_v04';
        , parameters: None
[2023-09-04T06:04:58.230+0000] {sql.py:427} INFO - Rows affected: 1
[2023-09-04T06:04:58.265+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v04, task_id=delete_data_from_table, execution_date=20220312T000000, start_date=20230904T060457, end_date=20230904T060458
[2023-09-04T06:04:58.356+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-04T06:04:58.634+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
